tests/ut/test_core_cloudfs.py

```diff
@@ -80,6 +81,35 @@ def test_mkfg_no_owner_raise(self):
             cloudfs.CloudFS(self._global_admin).mkfg(self._name)
         self.assertEqual(error_message, error.exception.message)
 
+    def test_mkfg_no_owner_fixedBlockSize(self):
+        self._init_global_admin(execute_response='Success')
+        ret = cloudfs.CloudFS(self._global_admin).mkfg(self._name, deduplication_method_type=self.fixed_block_size)
+        self._global_admin.get.assert_not_called()
+        self._global_admin.execute.assert_called_once_with('', 'createFolderGroup', mock.ANY)
+        expected_param = self._get_mkfg_object(fixed_block_size=True)
+        actual_param = self._global_admin.execute.call_args[0][2]
+        self._assert_equal_objects(actual_param, expected_param)
+        self.assertEqual(ret, 'Success')
+
+    def test_mkfg_with_local_owner_fixedBlockSize(self):
+        self._init_global_admin(execute_response='Success')
+        self._mock_get_user_base_object_ref()
+        ret = cloudfs.CloudFS(self._global_admin).mkfg(self._name, self._local_user_account, self.fixed_block_size)
```

Comment:
When passing named arguments, pass them as such even if they are at the right position. I.e. user=self._local_user_account